services/components/select.js

```diff
@@ -431,14 +433,17 @@ function handler(el, options) {
 
     // add selector to the component el, BEHIND the component's children
     dom.prependChild(el, selector);
+    window.kiln.trigger('add-selector', el, options);
```

Comment:
We should pass along `parent` as well. Spaces needs the context of the parent and what is in that component list to function properly.